repos/rust-lang/chalk.toml

```diff
@@ -0,0 +1,10 @@
+org = "rust-lang"
+name = "chalk"
+description = "An implementation and definition of the Rust trait system using a PROLOG-like logic solver"
+bots = ["rustbot", "bors"]
+
+[access.teams]
+types = "write"
```

Comment:
Is compiler and lang team access no longer needed?

Reply:
See https://github.com/rust-lang/team/pull/1185#issuecomment-1884017701.